client/etcd/client.go

```diff
@@ -144,60 +144,67 @@ func (c *csclient) createServices(opts services.OverrideOptions) (services.Servi
 }
 
 func (c *csclient) createTxnStore(opts kv.OverrideOptions) (kv.TxnStore, error) {
+	// validate the override options because they are user supplied
```

Comment:
nit: period at the end of each comment

Reply:
It looks like we do this for function comments but not inline comments (i.e. [here](https://github.com/m3db/m3cluster/blob/d971450316604c151719f53afbb95539e6dbafbb/client/etcd/client.go#L52) and [here](https://github.com/m3db/m3cluster/blob/d971450316604c151719f53afbb95539e6dbafbb/client/etcd/client.go#L311) in this file). I'm inclined to stick to that convention

Reply:
That was before we started to enforce the period thing, I think going forward for all the new comments we should always add a period.